spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkSessionCatalog.java

```diff
@@ -397,4 +401,61 @@ public UnboundFunction loadFunction(Identifier ident) throws NoSuchFunctionExcep
       return getSessionCatalog().loadFunction(ident);
     }
   }
+
+  @Override
+  public View replaceView(
+      Identifier ident,
+      String sql,
+      String currentCatalog,
+      String[] currentNamespace,
+      StructType schema,
+      String[] queryColumnNames,
+      String[] columnAliases,
+      String[] columnComments,
+      Map<String, String> properties)
+      throws NoSuchViewException, NoSuchNamespaceException {
+    throw new UnsupportedOperationException("SessionCatalog doesn't support views");
```

Comment:
All these methods will later be implemented form https://github.com/apache/iceberg/pull/10231

Currently not required for adding a procedure for view.